packages/web-components/fast-components/src/avatar/avatar.styles.ts

```diff
@@ -0,0 +1,66 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-foundation";
+import {
+  accentFillRestBehavior,
+  heightNumber,	
+  neutralForegroundRestBehavior
+} from "../styles/index";
+
+export const AvatarStyles = css`
+  ${display("flex")} 
+  :host {
+    max-width: calc(${heightNumber} * 2.5px);
+    height: calc(${heightNumber} * 2.5px);
+    position: relative;
+  }
+  
+  .link {
+    text-decoration: none;
+    color: ${neutralForegroundRestBehavior.var};
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    min-width: 100%;
+  }
+
+  .square {
+    border-radius: calc(var(--corner-radius) * 1px);
+    min-width: 100%;
+    overflow: hidden;
+  }
+
+  .coin {
+    position: relative;
+    display: flex;
+  }
+
+  .circle {
+    border-radius: 100%;
+    min-width: 100%;
+    overflow: hidden;
+  }
+
+  .image {
+    min-width: 100%;
+    position: absolute;
+    display: block;
+  }
+
+  .name {
+    font-size: var(--type-ramp-plus-5-font-size);
+    line-height: var(--type-ramp-plus-5-line-height);
+    display: block;
+  }
+
+  ::slotted(fast-badge) {
+    --badge-fill-primary: #00FF00;
```

Comment:
Let's go with `accent` here if we're going to add a default slotted value - I know we've used "primary" in docs, but we use accent in practice for our appearances. We likely should do so here as well.

Reply:
Do we have a set map here as well? I noticed this is a fairly bright green - just curious of the use case.